documentation/assemblies/deploying/assembly-deploy-create-cluster.adoc

```diff
@@ -15,7 +15,7 @@ This applies, for example, to a Kafka cluster running outside of Kubernetes.
 But you can deploy and use the Topic Operator and User Operator as standalone components.
 
 NOTE: The Cluster Operator can watch one, multiple, or all namespaces in a Kubernetes cluster.
-The Topic Operator and User Operator watch for `KafkaTopics` and `KafkaUsers` in the single namespace of the Kafka cluster deployment.
+The Topic Operator and User Operator watch for `KafkaTopic` and `KafkaUser` resources in the single namespace of a Kafka cluster deployment.
```

Comment:
Since the namespace can be configured, maybe this is more precise?

```suggestion
The Topic Operator and User Operator watch for `KafkaTopic` and `KafkaUser` resources in a single namespace.
```